modules/exploits/windows/iis/iis_webdav_upload_asp.rb

```diff
@@ -118,7 +157,9 @@ def exploit
       'method'       => 'GET'
     }, 20)
 
-    if (! res)
+    sleep(2)
```

Comment:
It was only done to keep the output in sync. Noticed at times, things got ahead of it self.
Unable to reproduce it on demand.